specs/latest/1.0/index.html

```diff
@@ -1979,6 +1981,33 @@ <h4>Attributes</h4>
             The actual height of the drawing buffer. May be different from the
             <code>height</code> attribute of the <code>HTMLCanvasElement</code> if
             the implementation is unable to satisfy the requested width or height.
+        <dt>
+            <code class=attribute-name>
+                <a id="DOM-WebGLRenderingContext-drawingBufferColorSpace">
+                    drawingBufferColorSpace
+                </a>
+            </code>
+            of type <code><a href="https://html.spec.whatwg.org/multipage/canvas.html#predefinedcolorspace">PredefinedColorSpace</a></code>
+            <a href="#refsPREDEFINEDCOLORSPACE">(specification)</a>
+        <dd>
+            The color space in which the drawing buffer's contents are assumed to
+            have been produced. Changing this attribute causes the drawing buffer to
```

Comment:
```suggestion
            The color space used to interpret the drawing buffer's content values. Changing this attribute causes the drawing buffer to
```

Reply:
We don't really care how it's "assumed to have been produced", just that this is how we should interpret the values.

Reply:
Good point and thanks for the suggestion. Have incorporated it.